Project.swift

```diff
@@ -29,6 +29,10 @@ func targets() -> [Target] {
                 .target(name: "TuistKit"),
                 .target(name: "ProjectDescription"),
                 .target(name: "ProjectAutomation"),
+                .external(name: "SwiftToolsSupport"),
```

Comment:
Were we importing those in code without explicitly declaring them here? I wonder if we can do something to prevent these things again.

Reply:
> I wonder if we can do something to prevent these things again.

We could scan for `import XXX` statements. Not sure if there's a better way 🤔